src/Libraries/Microsoft.Extensions.Resilience/Resilience/Internal/ResilienceDimensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.Resilience.Resilience.Internal;
+
+internal static class ResilienceDimensions
+{
+    public const string FailureSource = "failure-source";
+
+    public const string FailureReason = "failure-reason";
+
+    public const string FailureSummary = "failure-summary";
+
+    public const string DependencyName = "dep-name";
+
+    public const string RequestName = "req-name";
```

Comment:
Any particular reason these are truncated names, rather than just the full words?

Reply:
Longer names lead to more data being send over the wire, so in general we try to use shorter ones when possible.